src/assets/mock-classes.ts

```diff
@@ -0,0 +1,15 @@
+import { Class } from './class';
```

Comment:
I think it is ok to use mock when you are coding. But shouldn't we have records in the database? We want to have these records in the database so that we can subscribe to them.

Reply:
A record of classes has been added into the database.